src/main/java/com/gmail/stepura/volodymyr/model/Review.java

```diff
@@ -0,0 +1,60 @@
+package com.gmail.stepura.volodymyr.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+@Entity(name = "reviews")
+public class Review {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @NonNull
+    @Column(name = "review_id")
+    private Long reviewId;
+
+    @ManyToOne
+    @NonNull
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @ManyToOne
+    @NonNull
+    @JoinColumn(name = "product_id")
+    private Product product;
+
+    @Column(name = "helpfulnessNumerator")
```

Comment:
```suggestion
    @Column(name = "helpfulness_numerator")
```
if the field name consists of one word, this annotation is redundant
if of several words - it should have the following format

Reply:
Changed to naming with two words.